src/JamesHalsall/Licenser/Licenser.php

```diff
@@ -210,4 +194,83 @@ private function log($message)
 
         $this->output->writeln($message);
     }
+
+    /**
+     * @param string  $path The path to the files/directory
+     */
+    public function check($path)
+    {
+        foreach ( $this->getFiles($path) as $file) {
+            $this->checkFile($file);
+        }
+    }
+
+    /**
+     * Checks the header of 1 file
+     * @param SplFileInfo $file
```

Comment:
should be a blank line after method description, `$file` parameter needs a description as well